README.md

```diff
@@ -30,10 +30,14 @@ milestones:                             # If there are no issues do not include
       cannot use the service reliably. This does not meet WCAG success criterion
       X.Y.Z (Criterion Description).
     date: 2020-09-30                    # The date that this issue will be fixed by
-serviceLastTestedDate: 2019-09-15       # In ISO format YYYY-MM-DD
+serviceLastTestedDate: 2019-09-15       # In ISO format YYYY-MM-DD. If you statement has not yet been tested, use 2020-11-30
 statementVisibility: public             # If set to public, the statement will be visible in production
 statementCreatedDate: 2019-09-30        # In ISO format YYYY-MM-DD
 statementLastUpdatedDate: 2019-09-30    # In ISO format YYYY-MM-DD
+automatedTestingOnly: true              # Only add this value if your service has only had automated testing. Otherwise, do not include
```

Comment:
Could we omit this property and just base it on whether they have provided additionalTestingDetails?

Reply:
Good question. My feeling is that making this explicit is easier for users to clearly understand their own YAML files, but happy to discuss.

Reply:
I prefer these things to be distinct - I can see the potential for additionalTestingDetails being used for other purposes in the future which would cause confusion.  To put it differently a service might end up with testing notes that have nothing to do with it being automatically tested.